### DISCUSSION TITLE
http://127.0.0.1:8000/posts/2019-07-28-lazy-proof-of-monty-hall/lazy-proof-of-monty-hall.html

### DISCUSSION THREAD

**Discussion by giscus** [laegsgaardTroels/laegsgaardTroels.github.io#26] [Announcements]
# http://127.0.0.1:8000/posts/2019-07-28-lazy-proof-of-monty-hall/lazy-proof-of-monty-hall.html

http://127.0.0.1:8000/posts/2019-07-28-lazy-proof-of-monty-hall/lazy-proof-of-monty-hall.html

**Comment by laegsgaardTroels** [comment]
Test!
